We don't want the new "stable" names, so if we just remove all the rules then
they don't happen.

diff -urN udev-208-org/rules/80-net-name-slot.rules udev-208/rules/80-net-name-slot.rules
--- udev-208-org/rules/80-net-name-slot.rules	2010-12-10 04:23:54.000000000 -0800
+++ udev-208/rules/80-net-name-slot.rules	2013-10-02 18:19:25.451793617 -0700
@@ -0,14 +0,1 @@
 # do not edit this file, it will be overwritten on update
-
-ACTION!="add", GOTO="net_name_slot_end"
-SUBSYSTEM!="net", GOTO="net_name_slot_end"
-NAME!="", GOTO="net_name_slot_end"
-
-IMPORT{cmdline}="net.ifnames"
-ENV{net.ifnames}=="0", GOTO="net_name_slot_end"
-
-NAME=="", ENV{ID_NET_NAME_ONBOARD}!="", NAME="$env{ID_NET_NAME_ONBOARD}"
-NAME=="", ENV{ID_NET_NAME_SLOT}!="", NAME="$env{ID_NET_NAME_SLOT}"
-NAME=="", ENV{ID_NET_NAME_PATH}!="", NAME="$env{ID_NET_NAME_PATH}"
-
-LABEL="net_name_slot_end"
